src/qt/rpcconsole.cpp

```diff
@@ -383,10 +384,13 @@ bool RPCConsole::RPCParseCommandLine(interfaces::Node* node, std::string &strRes
     }
 }
 
-void RPCExecutor::request(const QString &command, const WalletModel* wallet_model)
+void RPCExecutor::request(QString command, const WalletModel* wallet_model)
 {
     try
     {
+        command.replace(QRegExp("@best"), "getbestblockhash()");
+        command.replace(QRegExp("@([\\d]+)"), "getblockhash(\\1)");
```

Comment:
The opening word boundary doesn't work with a "word" that starts with the `@` character. But the closing word boundary indeed improves behavior and error handling:
```suggestion
        command.replace(QRegExp("@best\\b"), "getbestblockhash()");
        command.replace(QRegExp("@([\\d]+)\\b"), "getblockhash(\\1)");
```